composer.json

```diff
@@ -20,7 +20,7 @@
         "php" : "~5.6|~7.0"
     },
     "require-dev": {
-        "phpunit/phpunit" : ">=5.4.3",
+        "phpunit/phpunit" : "^5.4.3|^6.5|^7.0",
```

Comment:
This is also wildly unsafe, as PHP <5.7 has no namespace support.

Reply:
Personally I'd go with PHPUnit 6.0 and PHP 7.0